src/api/resources/transactions.js

```diff
@@ -29,10 +29,5 @@ export default class TransactionsResource extends APIResource {
 		this.create = apiMethod({
```

Comment:
Should this be `create` or `broadcast`? Same applies to signatures. @Tosch110

Reply:
good point! When it only sends (broadcasts) the transaction and signature I find `broadcast` better. It does not create anything.

Reply:
I think `API` wise it's attempting to create.
`broadcast` itself is done by the nodes

Reply:
The confusion already starts. I think this is `broadcast` rather, we provide the already created transaction/signature and `broadcast` it to the node with this API.